syft/execution/role.py

```diff
@@ -5,8 +5,7 @@
 
 import copy
 
-# TODO torch shouldn't be used here
-import torch
+from syft.generic.frameworks.types import framework_packages
```

Comment:
A bit weird to import this from a module called `types` isn't it?

Reply:
Yeah, it is. That's where the dependency checks for available imports happen though. We could import it in the `__init__.py`, I suppose.